docs/update_bios.rst

```diff
@@ -0,0 +1,52 @@
+BIOS Updates
+============
+
+Below are the steps for updating the BIOS on the app and mon servers. You can also follow these steps to update the BIOS on your computer, such as the Admin Workstation Computer.
+
+What you need
+~~~~~~~~~~~~~
+
+  #. A clean USB device to download the ``.bio`` file
+  #. An internet-connected workstation, such as the *Admin Workstation*
+  #. A UPS (uninterrupted power supply), such as a surge-protecting power supply with a backup battery
+  #. A keyboard and mouse to plug into the servers (you can skip this if you are updating a laptop.)
+
+Perform Backups
+~~~~~~~~~~~~~~~
+
+Before performing any updates on the servers, we recommend you :doc:`back up the app server <backup_and_restore>`.
+
+Preparing the USB Stick
+~~~~~~~~~~~~~~~~~~~~~~~
+
+Using the Disks program, delete existing partitions on the USB device, if applicable, and reformat the entire device with one FAT32 partition. You do not need to set a passphrase or encryption on this device. Note that you will lose access to all existing data on this USB stick.
+
+Download and Verify Appropriate .bio Files
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+  .. note:: We will be following the F7 BIOS update method, `as documented by Intel (available as a PDF)`_.
+
+Check the make and model of your servers and proceed to the Intel Download Centre to download the correct file. Each make and model of Intel NUC will offer various file types; select the ``.bio`` option.
```

Comment:
Would it be worth linking directly to the Download Center section for our recommended hardware?